### ISSUE TITLE
[UI] Istio Traffic Graph - When users click on hyperlinks in iframe they can't navigate back to traffic graph

### ISSUE THREAD

**Issue by davidnuzik** [rancher/rancher#21811]
Version: master-head (v2.3) (7/26/19)

**What kind of request is this (question/bug/enhancement/feature request):**
Bug

**Steps to reproduce (least amount of steps as possible):**
- Set up istio and any workloads/services so we have content in Traffic Graph
- On the Traffic Graph (the iframe) click on one of the links at the top right (such as any of the three pictured below)
![image](https://user-images.githubusercontent.com/45179589/61984589-df334300-afb9-11e9-9d76-0a9fed3b9f7e.png)

**Result:**
Now I can't get back to the traffic graph page. I have to refresh the page or navigate away and come back. It looks like my browser back arrow also works, but this wasn't very clear to me initially (since it's in an iframe).

Do we want to possibly add a link or something that let's us go back to the traffic graph in the iframe? If so, how can we do this and make it look good. We may want to consider this.

**Comment by loganhz** [comment]
Yes, it's not good. But it looks like there is nothing we can do for it when using iframe. cc @vincent99 @westlywright

**Comment by davidnuzik** [comment]
What about including the small nav directory they have? Is this not possible?
![image](https://user-images.githubusercontent.com/45179589/62247426-ae2c8700-b39a-11e9-89de-3c4e45046eb7.png)

**Comment by loganhz** [comment]
I think we cannot control the style for that nav in Kiali UI.

**Comment by tfiduccia** [comment]
@loganhz - Can we hide the menu on the right?
![image](https://user-images.githubusercontent.com/11514927/62901975-048eb380-bd13-11e9-9626-7d1bca3605db.png)

**Comment by vincent99** [comment]
That's a section of the UI that's being hidden with negative offsets on the iframe.  You can show it, but it doesn't really solve the problem of leaving you stranded places because you also need the left-nav in some places.  And I think it exposes the Namespace dropdown with all of them in it.

The right side can't just be chopped off, at best you could watch for page changes (since this is inside the same domain) and remove the element from the DOM.

Seems like this should just open in a new window instead of being iframed at all, I'm not sure what we're gaining here.

**Comment by tfiduccia** [comment]
@logan - as we discussed let's add a "Refresh Graph" button next to the Namespace drop-down.
![image](https://user-images.githubusercontent.com/11514927/62905865-2e4ed700-bd21-11e9-8990-d182dfc24913.png)

**Comment by tfiduccia** [comment]
@davidnuzik - button is in
![image](https://user-images.githubusercontent.com/11514927/62987985-73ddd380-bdf7-11e9-99a0-10d9c2f5e609.png)

**Comment by davidnuzik** [comment]
Version: master-head (v2.3) (8/14/19)

I can confirm the "Refresh Graph" button is in now. This effectively lets us either refresh the iframe from the graph view, or if we have navigated away from the graph essentially navigate back to the Kiali graph. In short, this more or less resolves the issue since we now have a way to get back to the graph via this button.

I ensured the button works as expected and brings us back to the correct location.
